tests/test_modeling_tf_wav2vec2.py

```diff
@@ -556,18 +556,17 @@ def test_inference_ctc_robust_batched(self):
     @require_pyctcdecode
     @require_librosa
     def test_wav2vec2_with_lm(self):
```

Comment:
We can't nicely load mp3 at the moment with `datasets` -> let's just use a `.wav` file for now